test/e2e/multikueue/e2e_test.go

```diff
@@ -19,6 +19,8 @@ package mke2e
 import (
 	"os/exec"
 
+	"sigs.k8s.io/kueue/test/util"
+
```

Comment:
the order should be 

- standard
- external
- internal

move this import back down